vista_2d/cell_sam_wrapper.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from segment_anything.build_sam import build_sam_vit_b
+from torch import nn
+from torch.nn import functional as F
+
+
+class CellSamWrapper(torch.nn.Module):
```

Comment:
Hi @finalelement, can we use the one in the core?
https://github.com/Project-MONAI/MONAI/blob/dbfe418c03073baf07a0e14cd7606571f3d0de18/monai/networks/nets/cell_sam_wrapper.py#L25

Reply:
Yes, we should show the usage of MONAI Core components.

Thanks.

Reply:
@KumoLiu @Nic-Ma Im unable to call the CellSamWrapper from the MONAI library. Im using the monai                           1.4.0rc6, is there another version of MONAI that I should be using for this development?

We can update the CellSamWrapper to this tutorial, once it's callable. I was checking the init.py file of networks, it seems like its missing from there.

Reply:
Please refer the usage here: https://github.com/Project-MONAI/model-zoo/blob/dev/models/vista2d/configs/inference.json#L23